include/http/router.js

```diff
@@ -145,23 +146,39 @@ module.exports = function (pb) {
                     .use(this.router.allowedMethods());
 
                 this._addDefaultMiddleware();
-
-                this.__server = this.app.listen(port, () => {
-                    pb.log.info('PencilBlue is ready!');
-                });
+                const config = {
+                    https: {
+                        port: 8080,
+                        options: {
+                            key: fs.readFileSync('../OAuthFlows/key.pem', 'utf8'),
+                            cert: fs.readFileSync('../OAuthFlows/server.crt', 'utf8')
+                        }
+                    }
+                };
+                const serverCallback = this.app.callback();
+                const httpsServer = https.createServer(config.https.options, serverCallback);
+                this.__server = httpsServer
+                    .listen(config.https.port, function(err) {
+                        if (!!err) {
+                            pb.log.info('PencilBlue is not ready!');
+                        } else {
+                            pb.log.info('PencilBlue is ready!');
+                        }
+                    });
+
+                // this.__server = this.app.listen(port, () => {
```

Comment:
can we removed this commented code?

Reply:
I just left it there because right now we're forcing the https server. All of this code is not going to master, is just going to the cms-koa branch.